specification/components/schemas/Address.yaml

```diff
@@ -1,27 +1,16 @@
 type: object
 description: An address associated with the patient.
+required:
+  - id
+  - use
 properties:
   id:
     type: string
     description: Unique system identifier for this address.
```

Comment:
Could the id section be extracted out as it seems to be mostly a replication with variations on language

Reply:
I can probably create a `Base` object - idea on a name?

Reply:
https://swagger.io/docs/specification/data-models/inheritance-and-polymorphism/